src/database/seeders/department.js

```diff
@@ -0,0 +1,17 @@
+
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert(
+    'Departments',
+    [
+      {
+        name: 'Software Engineering',
+      },
+      {
+        name: 'Accounting',
+      }
+    ],
+    {}
+  ),
+
+  down: (queryInterface, Sequelize) => queryInterface.bulkDelete('Department', null, {})
```

Comment:
Your value for the table name for the up and down methods don't tally. You have `Department` in the down and `Departments` in the up method.

Reply:
Corrected, Please review